pkg/messaging/v1/invoke_method_response.go

```diff
@@ -54,7 +54,9 @@ func (imr *InvokeMethodResponse) WithRawData(data []byte, contentType string) *I
 	}
 
 	imr.r.Message.ContentType = contentType
-	imr.r.Message.Data = &any.Any{Value: data}
+
+	// Clone data to prevent GC from deallocating data
+	imr.r.Message.Data = &any.Any{Value: cloneBytes(data)}
```

Comment:
This seems like a workaround, underlying problem could be something else in http server, during normal call flow, the go routine started by http server for this call would not have finished and it would not GC. Is it a bug in the http server implementation which we are using and is causing this.